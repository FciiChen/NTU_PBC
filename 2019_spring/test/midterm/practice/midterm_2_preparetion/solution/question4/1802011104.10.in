夜色深黑，層雲飛動，銀蛇般的閃電，灼亮暗金色的雲層邊緣，將十萬里漠漠長空，犁出阡陌縱橫。

　　一個黑雲壓城暴雨欲來之夜。

　　「嚓！」

　　一聲暴雷終於劃裂夜的寂靜，天地瞬間白茫茫大亮，勾勒大地之上樹木張牙舞爪的猙獰黑影，在那些長而妖亂的樹影之間，有數條更黑的影子，流星般飛越。

　　當先一人輕功卓絕，身形快得幾乎生出淡淡虛影，只是每次落地時，似乎都有些踉蹌，看那姿態似乎氣力不濟，然而每次將要栽落時，那人都順勢一扭身，更快更猛的射出去，絲毫不顧惜氣力，絲毫不給自己停頓的機會。

　　那人身子微微前傾，一個狂奔時最省力的姿勢，雙手卻緊緊抱住了懷中的一個小小包裹。

　　那小小一團護在他懷中，風雨不驚，那人前奔時猶自不忘用手護著，唯恐沾著一星雨絲。

　　他身後，幾條人影不即不離，以護衛的姿勢跟隨著，幾個人輕功雖有高下之分，但步姿頻率一致，圍護的方式十分有章法，一看便知道訓練有素，除了最前面那人埋頭前奔之外，後面幾人疾行中猶自不斷回頭，似乎在注意著身後的動向。

　　隆隆雨聲隔絕喧囂，狂暴的風卻將身後一些隱隱的動靜捲了來——馬蹄踏在水窪中的聲音、刀劍摩擦交擊的聲音、長鞭焦躁頻頻抽打在馬身的聲音。


 
　　這些聲音傳入這個疾奔的小小隊伍耳中，這些疲憊而狼狽的人們腳下更快。

　　很明顯，這是一場雨夜追殺，在蜿蜒山路和蒼青密林間，在惡劣天氣下，追逐者和逃亡者，進行著體力和耐力的比拚。

　　「好歹快到地頭了！」逃亡者隊伍中，一個魁梧大漢抹一把雨水，翹首望向蒼山背後某個方向，滿是血絲的眼底，閃爍起希望的星火。

　　「等到了，趕緊看看小六的傷。」另一個頎長玉面男子轉過頭，目光關切的看著身後一個持雙劍的少年。

　　那個叫小六的，看起來還是孩子，蒼白清瘦，遍身血染，面對幾人齊齊看過來的關心眼光，倔強的抿著唇，搖搖頭。

　　「叫你別來你非要來，這下好了，拖後腿！」一個矮個子男子斜著嘴角，睨視著那瘦弱少年，卻順手彈出一顆藥丸，塞到那少年嘴裡。

　　那少年呸的一口將藥吐在塵埃。

　　「你！」

　　「三虎！」抱著包裹的領頭男子沉聲一喝，矮子立即住嘴扭過頭去，領頭男子目光有些歉疚的看著這個少年……小六還未學成，本不該走這一趟，可是……他嘆息一聲，摸摸那少年的頭，道：「好在快到了……」


 
　　「咻！」

　　猛烈的破空風聲穿透雨幕，剎那間截斷他的語聲，雨花伴著血花濺起，奔在最後負責警戒的一個身影踉蹌一下，無聲栽落。

　　透過他後背的森黑的鋒尖，將這群逃亡者眉宇間剛露出的喜色釘住！

　　敵人追來了！

　　領頭那人下意識緊了緊懷中包裹，抿了抿唇，一甩頭間滿身雨水飛散，濕漉漉的臉倒映在閃電的白光裡，眼神隼利如鷹掠向隊伍之末。

　　接收到他眼神的魁梧大漢霍然扭身，大笑道：「奶奶的，事兒真多！」掌間青光一亮，二話不說撲向追逐者。

　　暴雨中粗豪冷笑聲釘子般射出，幾乎剛落地那一刻，那個看起來已經筋疲力盡的大漢，便手起刀落，連殺數人，倒落的敵人屍體將道路阻住。

　　被激怒的敵人包抄上來，將他圍在中間，雨水沖刷出廝殺者的輪廓，泥濘裡響起不知是誰的嘶吼，大片大片血花混雜著雨水潑灑而開，將蒼白的閃電染紅。

　　閃電裡黑色背影孤獨的留在雨幕那頭，以一己之力死死擋住敵人前進的步伐。而這一頭，其餘人連猶豫都沒有，咬牙頭也不回繼續前行。


 
　　沒有時間猶豫，更沒有時間傷心，這樣的場景，在那白骨鮮血鋪就的逃亡之路上已經綿延了一地，一路上，三百人的隊伍，便是生生以這樣的方式，被削薄成今夜最後剩下的寥寥數人。

　　沒有人不滿，更沒有人畏怯，這是他們存在的全部使命——六百年前驚才絕豔的皇者，創立一代代被大力培養的密衛，這些人享有最高等級供奉，家族妻兒都被專門照拂，平時不作戰，不護衛，不被任何達官貴吏驅使，一生也許都未必派上一次用場，然而一旦用上他們，便是天地傾覆之刻，那麼到時，人人都是以一當百的死士！

　　何止以一當百？長達千里的逃亡之路，面對數萬不死不休追逐的大軍，暗殺、設伏、反間、攻防……出發時三百人，到了這裡只剩下最後五人，然而，換來的卻是數千敵人屍首，一路倒伏。

　　在重門深鎖的皇家密檔裡，他們被稱作：血浮屠！

　　然而，正如血浮屠永不能為世人所知一般，屬於這支精兵隊伍再輝煌的戰績，都將注定被歷史無聲淹沒。

　　存在，就是為了在需要的時刻，犧牲。

　　身後敵人的喧囂再次傳來，一條命只能拖延寶貴的一刻，小六眼神一冷，返身要撲，矮子三虎突然伸手將他狠狠一拽，拽到一邊。

　　「逞能！」

　　暴雨裡三虎束緊腰，那裡有個一直流血的傷口，很不滿的道：「我就知道好事該輪到我了。」


 
　　他倒拖著刀轉過身去，留給同伴一個懶洋洋的背影，揮揮手。

　　「如果誰活下來，記得告訴我女兒，她爹再也娶不了二房了，叫她放心！」

　　剩下的三個人沉默著，小六臉色更白，領頭男子閉了閉眼。

　　「好！」

　　廝殺聲遠遠拋在身後，三個人拚命飛馳，這是拿命博來的時間，沒有誰有權利浪費！

　　遠處傳來一聲淒厲而熟悉的嘶吼，尖利的穿透天地喧囂，領頭男子立即道：「別回頭！」

　　然而小六已經回過頭去，一轉首間看清身後骨肉飛灑踐踏成泥一幕，眼色血紅。

　　隨即他無聲無息撲了回去。

　　領頭男子一伸手便抓住了他，小六死命掙扎，卡在臂上的手卻鐵鉗般動也不動，雨聲中聽見老大清晰穩定的道：「阿衍，你去！」


 
　　小六霍然回首，怒道：「老大，你瘋了！」

　　那頎長男子已經笑笑，道：「我家孩兒，拜託老大。」

　　領頭男子默然點頭，掉轉目光，小六還要說什麼，卻立即被封了啞穴。

　　頎長男子摸摸他的頭，笑容溫暖，道：「小六，天戰世家如今只剩下你一個傳人，你好好活著。」
BREAK