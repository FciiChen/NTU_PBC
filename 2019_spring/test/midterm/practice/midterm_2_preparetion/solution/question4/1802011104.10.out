領_男 6
頭_子 6
三_人 4
一_雨 3
的_個 3
的_著 3
的_音 3
看_來 3
逃_者 3
過_去 3
